MongoDB/containers/0.1.0/mongodb-config/connect.sh

```diff
@@ -3,11 +3,11 @@ DIG=/opt/rancher/bin/dig
 
 function cluster_init {
 	sleep 10
-	MYIP=$(ip -o -4 addr list eth0 | awk '{print $4}' | cut -d/ -f1 |  sed -n 2p)
+	MYIP=$(ip -o -4 addr list eth0 | awk '{print $4}' | cut -d/ -f1)
```

Comment:
if you use giddyup ip myip in a new'ish version it returns the container ip from metadata.